code/game/machinery/atmoalter/canister.dm

```diff
@@ -420,9 +421,9 @@ update_flag
 
 	data["hasHoldingTank"] = holding ? 1 : 0
 	if(holding)
-		data["holdingTank"] = list("name" = holding.name, "tankPressure" = round(holding.air_contents.return_pressure()))
-
-	return data
+		data["holdingTank"] = list("name" = holding.name, "tankPressure" = round(holding.air_contents.return_pressure()))
+
+	return data

```

Comment:
not sure either, maybe the new line threw something off with github's change detection